spec/features/rendering_spec.rb

```diff
@@ -64,4 +64,50 @@ def index
       expect(subject.cache_key).to eq([:value, 'the block'])
     end
   end
+
+  describe 'attributes with default value' do
+    let(:component_class) do
+      Class.new(Components::Rails::Component) do
+        defaults(some_attribute: 'default value')
```

Comment:
@joeosburn-desiringgod is this what you mean by "check out the spec"?

Reply:
@scottcorgan Yes, that's part of it. More generally, I mean look at all of the additional spec code added, i.e: https://github.com/desiringgod/components-rails/blob/adec2cc67eba028a048f74661fa0b21a0ea0aec7/spec/features/rendering_spec.rb#L68-L112